.github/workflows/esm-lint.yml

```diff
@@ -63,14 +62,23 @@ jobs:
       - run: npm install ./artifact
       - run: npx snowpack@2 build
       - run: cat build/web_modules/$NPM_MODULE_NAME.js
+  TypeScript:
+    runs-on: ubuntu-latest
+    needs: Pack
+    steps:
+      - uses: actions/download-artifact@v2
+      - run: npm install ./artifact
+      - run: echo "${{ env.IMPORT_TEXT }} '${{ env.NPM_MODULE_NAME }}'" > index.ts
+      - run: tsc index.ts
+      - run: cat index.js
   Node:
     runs-on: ubuntu-latest
     needs: Pack
     steps:
       - uses: actions/download-artifact@v2
       - uses: actions/setup-node@v1
         with:
-          node-version: 13.x
+          node-version: 14.x
       - run: echo "${{ env.IMPORT_TEXT }} '${{ env.NPM_MODULE_NAME }}'" > index.mjs
-      - run: npm install ./artifact svg-tag-names
+      - run: npm install ./artifact
```

Comment:
By the way this failed because #83 broke rollup and svg-tag-names was not being bundled anymore

Reply:
That only breaks the CI, not the actual build.

Reply:
```js
git checkout ae3ee00
npm install
npm run build
```

you'll see:
```
index.ts → ....
(!) Unresolved dependencies
https://rollupjs.org/guide/en/#warning-treating-module-as-external-dependency
svg-tag-names (imported by index.ts)
created . in 1.9s
```

now open `index.js` and you'll see that `import { svgTagNames } from 'svg-tag-names';` is still there, even though 'svg-tag-names' is not a dependency… hence it broke the build 😃